docs/ui-coverage/results-api.mdx

```diff
@@ -311,8 +311,8 @@ jobs:
           TERM: xterm
           CYPRESS_RECORD_KEY: $(CYPRESS_RECORD_KEY)
 
-<strong>+     - script: |
-</strong>+           npm install --force https://cdn.cypress.io/extract-cloud-results/v1/extract-cloud-results.tgz
++     - script: |
++           npm install --force https://cdn.cypress.io/extract-cloud-results/v1/extract-cloud-results.tgz
 +           node ./scripts/verifyUICoverageResults.js
 +       displayName: 'Verify UI Coverage Results'
 +       env:
```

Comment:
@emilyrohrbough Yah unfortunately the styles for code previews with diff in the docs is not great - this is probably the one loss of the transfer. It's possible to pull in styles but it because more work than I could tackle at the time.